continuum/tasks/utils.py

```diff
@@ -24,3 +25,33 @@ def split_train_val(dataset: TaskSet, val_split: float = 0.1) -> Tuple[TaskSet,
     val_dataset = TaskSet(x_val, y_val, t_val, dataset.trsf, dataset.data_type)
 
     return train_dataset, val_dataset
+
+
+def concat(dataset: TaskSet, *others: TaskSet) -> TaskSet:
+    """Concatenate a dataset A with one or many *other* datasets.
```

Comment:
Concat pourrait prendre une liste de taskset pour etre plus general, comme pour numpy, et concatener tout

Reply:
You're right, I've changed it